magento/magento2
# [MFTF] Refactoring of AdminUpdateSimpleProductWithRegularPriceInStockEnabledFlatTest

### Description (*)
The test is refactored according to the best practices

### Manual testing scenarios (*)
1. Enable flat catalog product
2. Create a simple product
3. Update product (main info) and save
4. Check if all changes are available on the Product Edit page in Admin
5. Check if all changes are available on the Product Detail page on Storefront
6. Check if search by the new SKU is successful


### Resolved issues:
1. [x] resolves magento/magento2#31234: [MFTF] Refactoring of AdminUpdateSimpleProductWithRegularPriceInStockEnabledFlatTest
